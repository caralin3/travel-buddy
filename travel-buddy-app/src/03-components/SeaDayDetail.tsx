import React from 'react';
import { Col, Row } from 'reactstrap';

export interface SeaDayDetailProps {
  day: number;
  description?: string;
}

export const SeaDayDetail: React.FC<SeaDayDetailProps> = ({ day, description }) => (
  <div className="port-detail">
    <Row>
      <Col className="port-detail__name text-primary">Day {day}: Sea Day</Col>
    </Row>
    <Row className="my-2">
      <Col xs={6} md={6}>
        <p className="port-detail__label">Notes</p>
        {description}
      </Col>
    </Row>
  </div>
);
