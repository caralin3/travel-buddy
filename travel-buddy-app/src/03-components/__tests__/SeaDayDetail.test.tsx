import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import { SeaDayDetail } from '../SeaDayDetail';

describe('SeaDayDetail', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<SeaDayDetail day={1} />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
