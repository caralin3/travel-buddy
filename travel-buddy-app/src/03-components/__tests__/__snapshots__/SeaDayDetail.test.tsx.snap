// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SeaDayDetail should render correctly 1`] = `
<div
  className="port-detail"
>
  <Row
    tag="div"
    widths={
      Array [
        "xs",
        "sm",
        "md",
        "lg",
        "xl",
        "xxl",
      ]
    }
  >
    <Col
      className="port-detail__name text-primary"
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
          "xxl",
        ]
      }
    >
      Day 
      1
      : Sea Day
    </Col>
  </Row>
  <Row
    className="my-2"
    tag="div"
    widths={
      Array [
        "xs",
        "sm",
        "md",
        "lg",
        "xl",
        "xxl",
      ]
    }
  >
    <Col
      md={6}
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
          "xxl",
        ]
      }
      xs={6}
    >
      <p
        className="port-detail__label"
      >
        Notes
      </p>
    </Col>
  </Row>
</div>
`;
